### ISSUE TITLE
#1 Accessibility: Field label is not visible

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#258]
On the news pages I'm getting an error for the News Archive drop down box "Field label is not visible"

Description of this issue:
The input field does not have a visible label indicating the purpose of the input field.
How to fix it:
A visible label helps with predictability for all types of users.

Thanks!
![capture](https://user-images.githubusercontent.com/15264315/40851782-646ffd20-6596-11e8-8c50-802dbbd7e42e.JPG)

**Comment by d4mation** [comment]
Turns out there technically _is_ a Label for Screen Readers. It just wasn't properly visible to them.

I have pushed a change to fix this to the Staging site (http://jacksoncollege.staging.wpengine.com/)

Let me know if it's good to push Live! 👍

**Comment by banksashleym** [comment]
This looks great! Good to push to live :)

**Comment by d4mation** [comment]
This change is now Live 👍
